entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+export GOOGLE_APPLICATION_CREDENTIALS="/usr/src/app/gcloud_key.json"
+gcloud config set project k8s-jkns-gke-soak
+gcloud config set account shanefu@k8s-jkns-gke-soak.iam.gserviceaccount.com
+gcloud config set compute/zone us-central1-f
+gcloud auth activate-service-account --key-file $GOOGLE_APPLICATION_CREDENTIALS
```

Comment:
Prefer quoting bracing variables (ref: https://google.github.io/styleguide/shellguide.html#variable-expansion)
```suggestion
gcloud auth activate-service-account --key-file "${GOOGLE_APPLICATION_CREDENTIALS}"
```

Reply:
done